src/onegov/search/indexer.py

```diff
@@ -421,18 +426,25 @@ def index(
                 (id_col := sqlalchemy
                     .column(id_key)),  # type: ignore[var-annotated]
                 sqlalchemy.column(self.TEXT_SEARCH_COLUMN_NAME),
+                sqlalchemy.column(self.TEXT_SEARCH_DATA_COLUMN_NAME),
                 schema=schema  # type: ignore
             )
             tsvector_expr = sqlalchemy.text(
                 'to_tsvector(:language, :data)').bindparams(
                 sqlalchemy.bindparam('language', type_=sqlalchemy.String),
                 sqlalchemy.bindparam('data', type_=sqlalchemy.JSON)
             )
+
             stmt = (
                 sqlalchemy.update(table)
                 .where(id_col == sqlalchemy.bindparam('_id'))
-                .values({self.TEXT_SEARCH_COLUMN_NAME: tsvector_expr})
+                .values({
+                    self.TEXT_SEARCH_COLUMN_NAME: tsvector_expr,
+                    self.TEXT_SEARCH_DATA_COLUMN_NAME:
+                    sqlalchemy.bindparam('data', type_=sqlalchemy.JSON)
```

Comment:
It might be better to use `JSONB` here and above, otherwise you introduce additional conversion overhead on Postgres' end since it has to convert from `::json` to `::jsonb`.

Reply:
done